### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by delsolsi** [DiscworldZA/gta-resources#303]
**Describe the bug**
Doesnt work with es_extended weight version, not using the latest one that removed es

**To Reproduce**
1. Install es_extended before the latest update that removes ES

error: https://pastebin.com/3E6k3qhF this causes inventory to bug out, giving items via /giveitem doesnt show in the inventory and the item being added or removed show as undefined.

**Comment by rex2630** [comment]
Yeah, maybe i should look on that when i will have more energy to do so

**Comment by delsolsi** [comment]
would really appreciate it if you find a fix <3

**Comment by adamsview** [comment]
@rex2630 if you could let us all know if you can resolve the weight issue that'll be fantastic, so far there's no other inventory... including inventory hud which works correctly, this is really annoying when working with properties, armouries and shops.

**Comment by rex2630** [comment]
Its not weight issue actually. Tell me what version of ESX broke it. Maybe that would be a lot  easier

**Comment by SomeThinGJ7** [comment]
the newest one breaks it @rex2630

**Comment by rex2630** [comment]
Can u send me a link for a exact commit, please?

**Comment by SomeThinGJ7** [comment]
https://github.com/ESX-Org/es_extended/releases/tag/1.2.0 @rex2630

**Comment by rex2630** [comment]
Are u sure it isnt this? :D https://github.com/ESX-Org/es_extended/commit/f236a67d63d2ec5979cff3d7ee12e7f3356b84e6

**Comment by SomeThinGJ7** [comment]
Yes thats it sorry.

**Comment by alfiansm** [comment]
Do you have solution? Please help me. I've got same error

**Comment by Bart-023** [comment]
Hello, first a great script, thans for that :)

I got also a esxItem error whit the last es_extended version.
Building a new server for our community.

Is to following error related to this or do i need to create an new report ?

SCRIPT ERROR: @disc-inventoryhud/server/inventory.lua:605: attempt to index a nil value (local 'esxItem')
> TriggerServerCallback (@es_extended/server/functions.lua:154)
> handler (@es_extended/server/common.lua:73)

Thanks in advance!

**Comment by rex2630** [comment]
getInventory function seems to be broken. v.name is sometimes nil. So look for that, i  still didn found anything

**Comment by BruhHoldMyGun** [comment]
> getInventory function seems to be broken. v.name is sometimes nil. So look for that, i still didn found anything

Is there a work around atm? Seeing as trying to access store inventories in my server isnt working?

**Comment by Bart-023** [comment]
The new es_extended is not supported, so you have to down grade you es_extended.

**Comment by BruhHoldMyGun** [comment]
> The new es_extended is not supported, so you have to down grade you es_extended.

mind linking a version that works?

**Comment by Bart-023** [comment]
> > The new es_extended is not supported, so you have to down grade you es_extended.
> 
> mind linking a version that works?

This one should work:
It’s not supported, try this one:

https://github.com/ESX-Org/es_extended/tree/130df3f18f0e248623060a87399ed710022d715a

**Comment by Bart-023** [comment]
> > The new es_extended is not supported, so you have to down grade you es_extended.
> 
> mind linking a version that works?

This one should work:

https://github.com/ESX-Org/es_extended/tree/130df3f18f0e248623060a87399ed710022d715a

**Comment by BruhHoldMyGun** [comment]
been getting this error even after adding in older es_extended. https://gyazo.com/dd14bebd8ca3d9cb5e894bdeef31689a

**Comment by arnevdaele** [comment]
Experiencing the same issue with weight system. Took a quick look into the code but didn't seem to find anything suspicious at this point. Would be nice if this could get fixed...

**Comment by arnevdaele** [comment]
> 
> 
> > > The new es_extended is not supported, so you have to down grade you es_extended.
> > 
> > 
> > mind linking a version that works?
> 
> This one should work:
> 
> https://github.com/ESX-Org/es_extended/tree/130df3f18f0e248623060a87399ed710022d715a

Indeed works with this version of ESX.

**Comment by Lars-Esx** [comment]
@arnevdaele can you send me a link cause the es_extended page is takendown

**Comment by arnevdaele** [comment]
> 
> 
> @arnevdaele can you send me a link cause the es_extended page is takendown

You can use the Zap-FiveM ES_Extended version included in this repo:
https://github.com/zap-fivem/esx

**Comment by thefrcrazy** [comment]
Hello, was there a fix?
![Screenshot_10](https://user-images.githubusercontent.com/37953895/82907059-6b115500-9f66-11ea-8bea-b5ddbe25f3d7.png)

**Comment by Marcox360** [comment]
> Hello, was there a fix?
> ![Screenshot_10](https://user-images.githubusercontent.com/37953895/82907059-6b115500-9f66-11ea-8bea-b5ddbe25f3d7.png)

I dont think so

**Comment by DiscordianRo0t** [comment]
**Issue: Have a fix for weight goes over max limit (even hard coded)

Searched for hours for a fix and all my items in database have the same weights as my config file.

**Comment by Demonith** [comment]
```
[script:disc-inventor] SCRIPT ERROR: @disc-inventoryhud/server/inventory.lua:605: attempt to index a nil value (local 'esxItem')
[script:disc-inventor] > TriggerServerCallback (@es_extended/server/functions.lua:160)
[script:disc-inventor] > handler (@es_extended/server/common.lua:71)
```
When i try to move item in inventory its get stuck and gray and cant use have to restart
